Modules/_interpretersmodule.c

```diff
@@ -841,17 +848,29 @@ Any keyword arguments are set on the corresponding config fields,\n\
 overriding the initial values.");
 
 
+/*[clinic input]
+_interpreters.create
+    config as configobj: object(py_default="'isolated'") = NULL
```

Comment:
Isn't the default value `None`?

Reply:
In the code the default is equivalent to None. I did not look deeper. @ericsnowcurrently?